src/components/Thumbnail/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import * as styles from './styles';
+
+const Thumbnail = props => {
+    let price = props.product.price;
```

Comment:
Talvez um deconstruct nos attrs

```suggestion
    const { price, specialPrice, rating } = props.product;
```